app/src/main/java/com/pac_man/Collectables/APowerUp.java

```diff
@@ -0,0 +1,10 @@
+package com.pac_man.Collectables;
+
+import com.bridge.renderHandler.sprite.Sprite;
+import com.pac_man.Collisions.ICollisionSubscriber;
+import com.pac_man.characters.Pacman.Block;
+
+public abstract class APowerUp implements ICollisionSubscriber {
+    protected Sprite sprite;
+    protected Block block;
```

Comment:
If i am not wrong the Block class does not exist anymore in the UML, It should be its interface IBlock. Also che if you have to implement that class or import it from the jar.

Reply:
I have changed that, Could you please review it again ?